packages/now-static-build/src/utils/_shared.ts

```diff
@@ -38,3 +50,75 @@ export async function writePackageJson(
     JSON.stringify(packageJson, null, 2)
   );
 }
+
+/**
+ * Reads the .vercel_build_output directory and returns and object
+ * that should be merged with the build outputs.
+ *
+ * At the moment only `functions/node` is supported.
+ */
+export async function readBuildOutputDirectory({
+  workPath,
+}: {
+  workPath: string;
+}) {
+  const output: { [key: string]: Lambda } = {};
+  const nodeFunctionPath = path.join(
+    workPath,
+    '.vercel_build_output',
+    'functions',
+    'node'
+  );
+  const nodeFunctionFiles = await glob('**/index.js', {
+    cwd: nodeFunctionPath,
+  });
+  const nodeBridgeData = await fs.readFile(path.join(__dirname, 'bridge.js'));
+
+  for (const fileName of Object.keys(nodeFunctionFiles)) {
+    const launcherFileName = '__now_launcher';
```

Comment:
```suggestion
    const launcherFileName = '___now_launcher';
```

nit: The underscores didnt match for these two files